apps/desktop/svelte.config.js

```diff
@@ -3,7 +3,7 @@ import { vitePreprocess } from '@sveltejs/vite-plugin-svelte';
 
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
-	preprocess: vitePreprocess(),
+	preprocess: vitePreprocess({ script: true }),
```

Comment:
Required in `vite-plugin-svelte@4.0.0+` in order to use enums in `.svelte` files